src/components/Facility/PatientNotesSlideover.tsx

```diff
@@ -111,35 +171,18 @@ export default function PatientNotesSlideover(props: PatientNotesProps) {
       message?.facility_id == facilityId &&
       message?.patient_id == patientId
     ) {
-      setReload(true);
+      refetchNotes();
     }
   });
 
   useEffect(() => {
-    async function fetchPatientName() {
-      if (patientId) {
-        const { data } = await request(routes.getPatient, {
-          pathParams: { id: patientId },
-        });
-        if (data) {
-          setPatientActive(data.is_active ?? true);
-        }
-      }
-    }
-    fetchPatientName();
-  }, [patientId]);
+    localStorage.setItem(localStorageKey, noteField);
+  }, [noteField, localStorageKey]);
```

Comment:
We might need this even when the component remounts. Doing this will completely remove the use case for local storage, right? correct me if am wrong .